meshtastic/__main__.py

```diff
@@ -395,6 +395,9 @@ def onConnected(interface):
                 )
 
         if args.sendping:
+            channelIndex = 0
+            if args.ch_index is not None:
+                channelIndex = int(args.ch_index)
```

Comment:
I see why you did this since it's copying what `sendtext` does below, but you probably shouldn't need to look at `args.ch_index` again here. Within `common()` we pull the channel index into our_globals. So I think this should be `channelIndex = our_globals.get_channel_index() or 0`

Reply:
Of course, I got it, I have been changing that.